index.html

```diff
@@ -121,6 +121,36 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+                <!-- ================================================ -->
+        <!-- ==================  TEMPLATE  ================== -->
+        <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
+
```

Comment:
Since these lines are just comments and make no sense when copied by every contributor, Please do remove these lines.  
Do not remove the next line, i.e, 
```
<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```
but enter your name inside the asterisks symbols just like below :)
```
<!-- ________ *Your_name* Contributor card START ________  -->
```